docs/whats-new/dotnet-docs-2021-06-01.md

```diff
@@ -0,0 +1,127 @@
+- [Coerce call arguments according to ECMA-33](../core/compatibility/jit/6.0/coerce-call-arguments-ecma-335.md)
+- [SYSLIB0025: SuppressIldasmAttribute is obsolete](../fundamentals/syslib-diagnostics/syslib0025.md)
+- [SYSLIB0026: X509Certificate and X509Certificate2 are immutable](../fundamentals/syslib-diagnostics/syslib0026.md)
+- [Blazor: Byte-array interop](../core/compatibility/aspnet-core/6.0/byte-array-interop.md)
+- [Upgrade Assistant telemetry](../core/porting/upgrade-assistant-telemetry.md)
+- [dotnet nuget trust](../core/tools/dotnet-nuget-trust.md)
+- [Deploy a Worker Service to Azure](../core/extensions/cloud-service.md)
+- [Diagnostics client library](../core/diagnostics/diagnostics-client-library.md)
+- [Microsoft.Diagnostics.NETCore.Client API](../core/diagnostics/microsoft-diagnostics-netcore-client.md)
+
+### Updated articles
+
+- [Encrypt XML elements with asymmetric keys](../standard/security/how-to-encrypt-xml-elements-with-asymmetric-keys.md) - Update how-to-encrypt-xml-elements-with-asymmetric-keys.md
+- [Platform compatibility analyzer](../standard/analyzers/platform-compat-analyzer.md) - Update CA1416 with custom guards and TFM target requires AssemblyInfo.cs enabled
+- [How to migrate from Newtonsoft.Json to System.Text.Json](../standard/serialization/system-text-json-migrate-from-newtonsoft-how-to.md) - Initial .NET 6 Preview updates for .System.Text.Json
+
+## Architecture guides
+
+### New articles
+
+- [The Dapr actors building block](../architecture/dapr-for-net-developers/actors.md)
+- [Traffic Control sample application](../architecture/dapr-for-net-developers/sample-application.md)
+
+### Updated articles
+
+- [The Dapr bindings building block](../architecture/dapr-for-net-developers/bindings.md) - Dapr Traffic Control
+- [The Dapr observability building block](../architecture/dapr-for-net-developers/observability.md) - Dapr Traffic Control
+- [Dapr reference application](../architecture/dapr-for-net-developers/reference-application.md) - Dapr Traffic Control
+- [The Dapr secrets management building block](../architecture/dapr-for-net-developers/secrets-management.md) - Dapr Traffic Control
+- [The Dapr service invocation building block](../architecture/dapr-for-net-developers/service-invocation.md) - Dapr Traffic Control
+
+## C# language
+
+### New articles
+
+- [Source Generators](../csharp/roslyn-sdk/source-generators-overview.md)
+
+### Updated articles
+
+- [Iterators](../csharp/iterators.md) - Describe async iterators
+- [Write safe and efficient C# code](../csharp/write-safe-efficient-code.md) - Edit pass, fix bugs, add nint and nuint
+
+## Community contributors
+
+The following people contributed to the .NET docs during this period. Thank you! Learn how to contribute by following the links under "Get involved" in the [what's new landing page](index.yml).
+
+- [tdykstra](https://github.com/tdykstra) - Tom Dykstra (18)
```

Comment:
Hmmm. It's not removing several FTEs this time.  I'll check the code again, and check review the list, making suggestions:

```suggestion
```